pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -35,36 +38,66 @@ import (
 type NodeClaim struct {
 	NodeClaimTemplate
 
-	Pods            []*v1.Pod
-	topology        *Topology
-	hostPortUsage   *scheduling.HostPortUsage
-	daemonResources v1.ResourceList
-	hostname        string
+	Pods                 []*corev1.Pod
+	reservedOfferings    map[string]cloudprovider.Offerings
```

Comment:
Discussed offline: Noting down that we should add a comment above this to note about why we are storing reservedOfferings in this way and calling FinalizeScheduling() with it rather than storing the data in labels as we schedule pods